hash/dc.xml

```diff
@@ -429,7 +427,8 @@ P830-****-** : Appears to be a Sega part number for promo discs.
 	-->
 
 	<software name="4wt">
```

Comment:
All the `hash/dc.xml` should be demoted to `supported="no"` by default. 
I'll eventually rig a regtest but until the list is incomplete that will never happen.